addons/website/views/snippets/s_adventure.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_adventure" name="Adventure">
+    <section class="s_adventure pt200 pb200 oe_img_bg" style="background-image: url('/web/image/website.s_adventure_default_image'); background-position: 50% 50%;">
```

Comment:
The background images are centered with the `o_bg_img_center` class.
So I think you can remove the inline style and add the class after `oe_img_bg`.